service.py

```diff
@@ -41,10 +41,10 @@ async def home():
 
 
 @app.get("/country/iso3")
-async def get_iso3(lat: float, lng: float) -> geocoding.Country:
```

Comment:
We still need to use the `geocoder with low tolerance` for reverse geocoding!

We should only use the `geocoder with high tolerance` for returning geometries.

Reply:
@tnagorra Agreed. It is now fixed.